src/components/ExpiryNotification/ExpiryNotificationModal.js

```diff
@@ -0,0 +1,138 @@
+import styled from '@emotion/styled'
+import { css } from 'emotion'
+import React, { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import { EmailComponent } from '@buidlhub/buidlhub-ens-notifications'
+
+import {
+  default as Button,
+  getButtonDefaultStyles,
+  getButtonStyles
+} from '../Forms/Button'
+import GlobalState from '../../globalState'
+import Loader from '../Loader'
+import Modal from '../Modal/Modal'
+import mq from 'mediaQuery'
+import { ReactComponent as Email } from '../Icons/Email.svg'
+import TruncatedAddress from './TruncatedAddress'
+
+const BUILDHUB_LINK =
+  'https://buidlhub.com/?utm_source=ens&utm_medium=link&utm_campaign=reminder'
+const EXPIRY_NOTIFICATION_MODAL_NAME = 'buidlhub-expiry-notification'
+const INPUT_NAME = 'buidlhub-email-input'
+
+// Custom components to match styling & use media queries
+const LoadingComponent = styled(Loader)`
+  display: inline-block;
+  margin: 0 10px;
+`
+
+const ModalIconContainer = styled('div')`
+  display: none;
+  justify-self: end;
+  ${mq.xLarge`
+    display: block;
+  `}
+`
+
+const FormComponent = styled('form')`
+  display: grid;
+  grid-gap: 10px;
+  grid-template-columns: 1fr;
+  ${mq.xLarge`
+    grid-template-columns: 2fr 1fr;
+  `}
+`
+
+const LabelComponent = ({ address, name }) => {
+  return (
+    <>
+      <div
+        className={css`
```

Comment:
Change to a styled-component

Reply:
Removed inline css & hoisted `font-weight` to parent styled component